src/components/routes/Error.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { SFC } from 'react';
+import FullHelmet from '../FullHelmet';
+import { Content, Title } from '../layout/StyledCopy';
+
+interface Props {
+    error: Error | null;
+}
+
+const Error: SFC<Props> = ({ error }) => (
+    <section>
+        <FullHelmet pageName="Error" />
+        <Title align={'center'}>Whoops!</Title>
+        {error && error.message && <Content align={'center'}>{error.message}</Content>}
```

Comment:
Wat ik altijd wel chill vind is als de env niet productie is dat hij ook de [stack trace](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack) laat zien op een error page. 
```jsx
{process.env.NODE_ENV !== 'production' && error.stack && <pre>{error.stack}</pre>}
```
Maar beslis zelf maar.